src/app/applications/english_auction/page.mdx

```diff
@@ -0,0 +1,449 @@
+export const metadata = {
+  title: "English Auction • Stylus by Example",
+  description:
+    "An example implementation of the English Auction in Rust using Arbitrum Stylus.",
+};
+
+{/* Begin Content */}
+
+# English Auction
+
+An Arbitrum Stylus version implementation of [solidity English Auction](https://solidity-by-example.org/app/english-auction/).
+
+## Auction
+- 1. Seller of NFT deploys this contract.
```

Comment:
I think you have to either use `-` or `1.`, otherwise nextjs gets confused and renders the [points with endlines](https://stylus-by-example-git-add-example-ui-offchain-labs.vercel.app/applications/english_auction). In your case, I think you meant to remove the `-`, right?